themes/startwords/assets/scss/article/_figure.scss

```diff
@@ -0,0 +1,29 @@
+/* figures (images, diagrams) in articles */
+
+body.article figure {
+    margin: rem(30px) 0;
+
+    img {
+        display: block;
+        margin: 0 auto;
+        
+        &.landscape {
+            width: 100%;
+            @media (min-width: $breakpoint-m) {
+                width: calc(100% + 200px);
+                margin-left: rem(-100px);
+                margin-right: rem(-100px);
+            }
+        }
+
+        &.portrait {
+            width: 75%;
```

Comment:
do you think it'll make much of a difference? it seems unlikely anyone would upload an extremely tall image, and even if that happened I'd think you'd want to avoid downscaling it vertically because the small details would be lost. I got this number from the designs, since the ratio between image width and container width is stable across devices.